hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsSqsConfiguration.kt

```diff
@@ -15,7 +15,7 @@ import org.springframework.jms.annotation.EnableJms
 @Configuration
 @EnableConfigurationProperties(HmppsSqsProperties::class)
 @EnableJms
-@AutoConfigureBefore(HealthEndpointAutoConfiguration::class)
+@AutoConfigureBefore(WebFluxAutoConfiguration::class)
```

Comment:
The async queue was being wired up after the Health Registry - something to do with the dependency graph between beans. So we need to jump in and create our beans as early as possible...